doc/sphinx/source/quickstart/installation.rst

```diff
@@ -633,6 +639,9 @@ the username and password can be omitted if they are not required. See e.g.
 `here <https://support.rstudio.com/hc/en-us/articles/200488488-Configuring-R-to-Use-an-HTTP-or-HTTPS-Proxy>`__
 for more information.
 
+HTTP connection errorrs (of e.g. type 404) to the Anaconda servers are rather common, and usually a retry
```

Comment:
It would be better to mention this in a new 'problem', because it's not related to proxies

Reply:
did I put it under the proxies issues? Yeah, lemme new problem it